rosdep/python.yaml

```diff
@@ -7271,6 +7271,10 @@ python3-myst-parser-pip:
   ubuntu:
     pip:
       packages: [myst-parser]
+python3-nanobind-pip:
```

Comment:
So it looks like this package is available on at least some distributions: https://pkgs.org/search/?q=nanobind .

So I'm going to suggest renaming this key to `python3-nanobind`, fill it in for the new distributions that have it, and fall back to the `pip` key for those that do not.  You won't be able to use this key to release things on the buildfarm, but at least that will be possible in the future.